themes/classic/templates/_partials/microdata/product-jsonld.tpl

```diff
@@ -0,0 +1,83 @@
+{**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License 3.0 (AFL-3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/AFL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
+ *}
+<script type="application/ld+json">
+  {
+    "@context": "https://schema.org/",
+    "@type": "Product",
+    "name": "{$product.name}",
+    "description": "{$page.meta.description|regex_replace:"/[\r\n]/" : " "}",
+    "category": "{$product.category_name}",
+    {if !empty($product.cover)}"image" :"{$product.cover.bySize.home_default.url}",{/if}
+    "sku": "{if $product.reference}{$product.reference}{else}{$product.id}{/if}",
+    "mpn": "{if $product.mpn}{$product.mpn}{elseif $product.reference}{$product.reference}{else}{$product.id}{/if}",
+    {if $product.ean13}"gtin13": "{$product.ean13}",{else if $product.upc}"gtin13": "{$product.upc}",{/if}
+    {if $product_manufacturer->name OR $shop.name}"brand": {
```

Comment:
I guess `$shop.name` always exists? 🤔

Reply:
It's really a fallback, if no manufacturer is defined... it seems that people hate to have Google Rich text warnings.. it's to avoid this situation and propose a clean but not perfect rich text test result (limit issues opened about this topic)... the people will be able to override the template if needed. 
We only need to be aligned with "what is a base version" => "less warning but not perfect" or "not complete, with more warnings, but no tweak" ;)

Reply:
@fdonnet ok, you're right 👍

Reply:
@fdonnet Shop name as manufacturer is a very good idea, if it's missing!!!